pipeline/detection.py

```diff
@@ -588,20 +599,26 @@ def _run_sextractor_once( self, image, apers=[5, ], psffile=None, tempname=None,
                                f"-------\nstdout:\n{res.stdout}" )
                 raise RuntimeError( f"Error return from source-extractor call" )
 
+            # Get the background from the xml file that sextractor wrote
+            sextrstat = votable.parse( tmpxml ).get_table_by_index( 1 )
+            bkg = sextrstat.array['Background_Mean'][0][0]
+            bkgsig = sextrstat.array['Background_StDev'][0][0]
```

Comment:
ugg I hate these command line utilities so much 🤮

Reply:
Just keep repeating to yourself : it's just an interface.  The disk file is a memory buffer.